tests/conftest.py

```diff
@@ -23,6 +23,27 @@
 pytest_plugins = ("celery.contrib.pytest",)
 
 
+@pytest.fixture(scope="module")
+def app_config(app_config):
+    """Override pytest-invenio app_config fixture.
+
+    Needed to set the fields on the custom fields schema.
+    """
+    app_config["RECORDS_RESOURCES_FILES_ALLOWED_DOMAINS"] = [
+        "inveniordm.test",
+    ]
+
+    app_config["FILES_REST_STORAGE_CLASS_LIST"] = {
```

Comment:
shelve/comment: we have to remember to configure this in `app-rdm` (and ZenodoRDM)